src/components/Main/components/CategoryCard.js

```diff
@@ -40,7 +38,7 @@ export const CategoryCard = ({category}) => {
       prevScreen: 'CategoryCard',
     })
   }
-  const getImg = () => api.host + category.img.formats.thumbnail.url;
+  const getImg = () => store.BASE_URL + category.img.formats.thumbnail.url;
```

Comment:
nothing is stopping you from keeping BASE_URL in .env https://www.npmjs.com/package/react-native-dotenv